CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing to Biconomy Projects 🚀
+
+First off, thank you for considering contributing to Biconomy! We truly appreciate your effort and contributions from the community are what make Biconomy awesome. 🙌
```

Comment:
I suggest dividing this into two:

 We truly appreciate your effort. Contributions from the community are what make Biconomy awesome 🙌